pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -48,10 +49,13 @@ var _ framework.EnqueueExtensions = &VolumeRestrictions{}
 const Name = names.VolumeRestrictions
 
 const (
+	preFilterStateKey = "PreFilter" + Name
 	// ErrReasonDiskConflict is used for NoDiskConflict predicate error.
 	ErrReasonDiskConflict = "node(s) had no available disk"
-	// ErrReasonReadWriteOncePodConflict is used when a pod is found using the same PVC with the ReadWriteOncePod access mode.
-	ErrReasonReadWriteOncePodConflict = "node has pod using PersistentVolumeClaim with the same name and ReadWriteOncePod access mode"
+	// ErrReasonReadWriteOncePodRepeat means that after the ReadWriteOncePod is turned on,
+	// the current cluster has nodes that meet the volume restrictions,
+	// so the pod will be scheduled to that node instead of this node.
+	ErrReasonReadWriteOncePodRepeat = "pod will be scheduled to nodes that meet volume restrictions"
```

Comment:
why did you change this?

This message should explain why the pod didn't schedule. This is not doing that.